src/hevm/src/EVM/SymExec.hs

```diff
@@ -406,7 +408,7 @@ reachableQueries = go []
           (go (PEq (Lit 1) c : pcs) t)
           (go (PEq (Lit 0) c : pcs) f)
         pure (tres <> fres)
-      _ -> pure [assertProps pcs]
+      _ -> pure [assertProps pcs mempty mempty]
```

Comment:
It's a little clunky to have to pass `mempty` twice to `checkSat'` / `assertProps` and friends. I wonder if there's some helper we can define that we expose as the main API from `EVM.SMT`?

Reply:
Done!